### PR TITLE
Mumble/forum bans

### PR DESCRIPTION
**Pull request by itsmartin**
Removed some information about emailing support which is no longer relevant, since mumble and forum bans are now handled through the usual appeal process.

### CODE REVIEW

**Comment by itsmartin** [comment]
Updated and slightly reworded in the process.

File: en.yml [hunk 1]
```diff
@@ -144,11 +144,10 @@ en:
...
                 - Persistent ban evasion will result in a permanent ban for all accounts.
-                - Players violating any forum rules may be banned from posting to the forums. If your ban stays for more than 7 days please contact support@oc.tc for the exact length of your ban.
+                - Players violating any forum rules may be banned from posting to the forums.
                 - Players violating any mumble rules may be kicked or banned from the mumble server.
                 - Players violating the rules on any Overcast Network service, including the game servers, the forums, and mumble, may be banned from any or all of the services.
-                - In game bans must be appealed by filing an appeal.
-                - Forum and mumble bans must be appealed via an email to support@oc.tc.
+                - All infractions may be appealed by filing an appeal.
```
Comment (tonybruess) [inline en.yml]: Should this say may or must?

Comment (itsmartin) [inline en.yml]: I think 'may' makes more sense. 'Must' almost makes it sound like appealing is compulsory for all infractions.

**Comment by tonybruess** [comment]
Merged: f663c51f83cabbcbf7c23cff2c7b9317897c951a

**Pull request closed without merging**